tests/Integration/Behaviour/Features/Scenario/Webservice/Endpoints/addresses.feature

```diff
@@ -70,7 +70,7 @@ Feature: Webservice key management
     And webservice key "ENABLEDENABLEDENABLEDENABLEDENAB" should have "PUT" permission for "addresses" resources
     And webservice key "ENABLEDENABLEDENABLEDENABLEDENAB" should have "DELETE" permission for "addresses" resources
     And I use Webservice with key "ENABLEDENABLEDENABLEDENABLEDENAB" to list "addresses"
-    Then I should get 5 items of type "address"
+    Then I should get 6 items of type "address"
```

Comment:
Just to be sure: is this because GDPR module creates some "empty address"?

Reply:
Affimative, It creates  a new customer with its adress.